### DISCUSSION TITLE
Example of CIRRUS_CRON

### DISCUSSION THREAD

**Discussion by 5u623l20** [cirruslabs/cirrus-ci-docs#1005] [Q&A] [answered]
Can anyone share an example of CIRRUS_CRON in a task?

@fkorotkov Is it possible to expand the docs for CIRRUS_CRON with an example?

**Comment by fkorotkov** [comment, marked as answer]
It seems the most common use case is to use it in `only_if` and other statements. For example, run some task only for cron invocations:

```yaml
task:
  name: Cron Specific Checks
  only_if: $CIRRUS_CRON != "" 
```

Also clarified a bit what's the value of `CRON_NAME` variable in https://github.com/cirruslabs/cirrus-ci-docs/pull/1006

More examples can be found in some of the [public repositories that use `CIRRUS_CRON`](https://github.com/search?l=YAML&q=CIRRUS_CRON&type=Code).
